apps/web/frontend/components/weekGrid/weekGridProjectRowGroup.tsx

```diff
@@ -56,7 +57,10 @@ export const WeekGridProjectRowGroup = ({
           role="cell"
         >
           <FaAngleRight className={`${isCollapsed ? '' : 'rotate-90'} transition`} />
-          {project.isArchived ? <span title="This project was archived">🗄️ {project.title}</span> : project.title}
+          <Link href={`projects/${project.id}`} className="flex items-center gap-1">
+            {project.isArchived ? <span title="This project was archived">🗄️ {project.title}</span> : project.title}
+            <FaArrowUpRightFromSquare />
```

Comment:
```suggestion
            <MdOpenInNew className="shrink-0" />
```
- They were using `<MdOpenInNew />` at BITBW. And the icon is smaller and the name tells what it does.
[https://react-icons.github.io/react-icons/search/#q=MdOpenInNew]()
- `shrink-0` // Prevents the arrow from shrinking

Reply:
Hmm, maybe we should ask @moncapitaine , since we've been using fa6 everywhere and it would make it inconsistent?